cime_config/testdefs/testlist_clm.xml

```diff
@@ -2084,6 +2084,16 @@
       <option name="comment"  >Also --nofireemis because this is a SP compset</option>
     </options>
   </test>
+  <test name="SMS_Ln9" grid="ne0POLARCAPne30x4_ne0POLARCAPne30x4_mt12" compset="IHistClm50Sp" testmods="clm/clm50cam6LndTuningMode_1979Start--clm/nofireemis">
+    <machines>
+      <machine name="derecho" compiler="intel" category="ctsm_sci"/>
```

Comment:
@ekluzek did you prefer to make these aux_clm tests instead of ctsm_sci?

Reply:
I think they should stay as ctsm_sci as these are special grids that we want to make sure works, but don't need to constantly test.